seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -988,36 +1031,70 @@
         "        currentNode.next = node;",
         "    }",
         "",
-        "    length++;",
+        "    this.length++;",
         "  }; ",
         "",
         "  this.remove = function(element){",
         "    // Only change code below this line",
         "",
         "    // Only change code above this line",
         "  };",
-        "};"
+        "} ",
+        "",
+        "// Test your code",
+        "var conga = new LinkedList();",
+        "conga.add('Kitten');",
+        "conga.add('Puppy');",
+        "console.log(conga.length);",
+        "conga.remove('Kitten');",
+        "console.log(conga.head);"
       ],
       "tests": [
         "assert((function(){var test = new LinkedList(); return (typeof test.remove === 'function')}()), 'message: Your <code>LinkedList</code> class should have a <code>remove</code> method.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.remove('cat'); return test.head().element === 'dog'}()), 'message: Your <code>Remove</code> method should reassign <code>head</code> to the second node when the first node is removed.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.remove('cat'); return test.length() === 1}()), 'message: Your <code>Remove</code> method should decrease the <code>length</code> of the linked list by one for every node removed.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog');test.add('kitten'); test.remove('dog'); return test.head().next.element === 'kitten'}()), 'message: Your <code>Remove</code> method should reassign the reference of the previous node of the removed node to the removed node's <code>next</code> reference ');"
+        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.remove('cat'); return test.head.element === 'dog'}()), 'message: Your <code>remove</code> method should reassign the <code>head</code> to the second node if the element to remove is the head.');",
+        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.remove('cat'); return test.length === 1}()), 'message: Your <code>Remove</code> method should decrease the <code>length</code> of the linked list by one for every node removed.');",
+        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog');test.add('kitten'); test.remove('dog'); return test.head.next.element === 'kitten'}()), 'message: Your <code>Remove</code> method should reassign the reference of the previous node of the removed node to the removed node's <code>next</code> reference ');"
       ],
       "solutions": [],
       "hints": [],
       "type": "waypoint",
       "challengeType": 1,
-      "translations": {}
+      "translations": {
```

Comment:
Please revert this line of change. We do not need to add empty translation blocks.